src/common/service/NetRemoteService.cxx

```diff
@@ -466,6 +467,39 @@ NetRemoteService::WifiAccessPointSetAuthenticationConfiguration([[maybe_unused]]
     return grpc::Status::OK;
 }
 
+using Microsoft::Net::Wifi::Dot11FrequencyBand;
+
+::grpc::Status
+NetRemoteService::WifiAccessPointSetFrequencyBands([[maybe_unused]] ::grpc::ServerContext* context, const ::Microsoft::Net::Remote::Wifi::WifiAccessPointSetFrequencyBandsRequest* request, ::Microsoft::Net::Remote::Wifi::WifiAccessPointSetFrequencyBandsResult* response)
+{
+    LOGD << std::format("Received WifiAccessPointSetFrequencyBands request for access point id {}", request->accesspointid());
+
+    WifiAccessPointOperationStatus status{};
+
+    auto frequencyBands = request->frequencybands();

```

Comment:
This is making a copy of the frequency bands `RepeatedField` structure. So, just need to make it a reference (and I think it can be `const`):

```suggestion
    const auto& frequencyBands = request->frequencybands();

```